prow-configurations/config.yaml.tpl

```diff
@@ -36,6 +37,20 @@ presubmits: # runs on PRs
     spec:
       containers:
       - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1 # created by running `docker build -t <image> .` in the integration-job directory.
+            
+postsubmits:
+  {{ .OrganizationOrUser }}/kyma:
+  - name: kyma-integration
+    branches:
+    - master
+    context: kyma-integration
+    skip_report: false # from documentation: SkipReport skips commenting and setting status on GitHub.
```

Comment:
```suggestion
    skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
    optional: true
```

Reply:
Same as above, postsubmits do not have `optional` or `skip_report` config